cmd/rootMain.go

```diff
@@ -124,12 +117,13 @@ func rootMain(cmd *cobra.Command, args []string) exitCode {
 		return exitCode{1}
 	}
 
-	mountpath, err := ioutil.TempDir(cachedir, "mnt")
+	// Create a temporary run space for aliases and server files.
+	rundir, err := ioutil.TempDir(cachedir, "run")
 	if err != nil {
-		cmdutil.ErrPrintf("Unable to create temporary mountpoint in %v: %v\n", cachedir, err)
+		cmdutil.ErrPrintf("Error creating temporary run location in %v: %v\n", cachedir, err)
 		return exitCode{1}
 	}
-	defer os.RemoveAll(mountpath)
+	defer os.RemoveAll(rundir)
```

Comment:
This seems to be interacting poorly with the mountpoint.

Reply:
What do you mean?

Reply:
I'm getting lots of "OSXFUSE Volume" icons on my desktop, which I think means they're not being unmounted correctly.